src/main/java/use_cases/display_curriculums/DisplayCurriculumsPresenter.java

```diff
@@ -2,6 +2,14 @@
 
 import java.util.List;
 
+/**
+ * A presenter class that provides output data to the dashboard
+ *
+ * Created: 11/30/2022
+ * Last updated: 12/01/2022
+ *
+ * @author Oswin Gan
+ */
```

Comment:
Good call on adding documentation! It's paramount that we assure all of our classes are properly documented.